packaging/prerelease/build_app.sh

```diff
@@ -63,26 +63,38 @@ if [ ! "$nopull" = "1" ]; then
   "$client_dir/packaging/check_status_and_pull.sh" "$updater_dir"
 fi
 
+echo "Loading release tool"
+"$client_dir/packaging/goinstall.sh" "github.com/keybase/release"
+release_bin="$GOPATH/bin/release"
+
+echo "Wait for CI"
+"$release_bin" wait-ci --repo=client --commit=`git -C $client_dir log -1 --pretty=format:%h`
+"$release_bin" wait-ci --repo=kbfs --commit=`git -C $kbfs_dir log -1 --pretty=format:%h`
+
 if [ -n "$client_commit" ]; then
-  cd "$client_dir"
-  client_branch=`git rev-parse --abbrev-ref HEAD`
-  function reset_client {
-    (cd "$client_dir" && git checkout $client_branch)
-  }
-  trap reset_client EXIT
-  echo "Checking out $client_commit on client"
-  git checkout "$client_commit"
+  (
+    cd "$client_dir"
+    client_branch=`git rev-parse --abbrev-ref HEAD`
+    function reset_client {
+      (cd "$client_dir" && git checkout $client_branch)
+    }
+    trap reset_client EXIT
```

Comment:
ACK! I should've caught this in my first pass. I think setting `trap foo EXIT` in a subshell will call `foo` when the *subshell* exits. Were you able to test this?